manage-data/ingest/transform-enrich/common-mistakes.md

```diff
@@ -14,33 +14,33 @@ There are many ways to achieve similar results when creating ingest pipelines, w
 This guide does not provide guidance on optimizing for ingest pipeline performance.
 :::
 
-## If Statements
+## Write concise `if` statements
 
-`if` statements are frequently used in ingest pipeline processors to ensure that a processor only runs when specific conditions are met. By adding an `if` condition, you can control when a processor is applied, making your pipelines more flexible and robust.
+Use `if` statements to ensure that an ingest pipeline processor is only applied when specific conditions are met.
 
-### Avoiding Excessive OR Conditions
+### Avoid excessive OR conditions
 
-When using the [boolean OR operator](elasticsearch://reference/scripting-languages/painless/painless-operators-boolean.md#boolean-or-operator) (`||`), it's easy for `if` conditions to become overly complex and difficult to maintain, especially when chaining many OR checks. Instead, consider using array-based checks like `.contains()` to simplify your logic and improve readability.
+When using the [boolean OR operator](elasticsearch://reference/scripting-languages/painless/painless-operators-boolean.md#boolean-or-operator) (`||`), `if` conditions can become unnecessarily complex and difficult to maintain, especially when chaining many OR checks. Instead, consider using array-based checks like `.contains()` to simplify your logic and improve readability.
 
-#### ![don't](../../images/icon-cross.svg) **Don't**: Run many ORs
+#### ![ ](../../images/icon-cross.svg) **Don't**: Run many ORs
 
 ```painless
 "if": "ctx?.kubernetes?.container?.name == 'admin' || ctx?.kubernetes?.container?.name == 'def'
 || ctx?.kubernetes?.container?.name == 'demo' || ctx?.kubernetes?.container?.name == 'acme'
 || ctx?.kubernetes?.container?.name == 'wonderful'
 ```
 
-#### ![do](../../images/icon-check.svg) **Do**: Use contains to compare
+#### ![ ](../../images/icon-check.svg) **Do**: Use contains to compare
 
 ```painless
-["admin","def", ...].contains(ctx.kubernetes?.container?.name)
+["admin","def","demo","acme","wonderful"].contains(ctx.kubernetes?.container?.name)
 ```
 
-A key implication is that using `["admin", "def", ...].contains(ctx.kubernetes?.container?.name)` checks for exact matches only. For example, `"admin".contains("admin")` returns `true`, but `"demo-admin-demo".contains("admin")` would not match unless you explicitly check for partial matches using something like `ctx.kubernetes.container.name.contains('admin') || ...`.
-
-Also, the null safe operator (`?.`) works as expected here—if the value is `null`, the `contains` method simply returns `false` without causing an error.
+:::{tip}
+This example only checks for exact matches. Do not use this approach if you need to check for partial matches.
+:::
```

Comment:
Remove unnecessary explanations. I don't _think_ you need to go into detail about checking for partial matches when that's not what the example is trying to illustrate. A quick `tip` would probably be sufficient. Let me know if you disagree.